src/components/AChat/AChat.vue

```diff
@@ -83,6 +83,11 @@ export default {
     currentScrollTop: 0,
     currentClientHeight: 0
   }),
+  computed: {
+    isWelcomeChat() {
+      return this.partners.map((item) => item.id).includes('chats.virtual.welcome_message_title')
```

Comment:
Can you reuse `isWelcomeChat` helper?

https://github.com/Adamant-im/adamant-im/blob/81ad0123e5897494a494b4ba6b0341ec10eb7947/src/lib/chat/meta/utils/isWelcomeChat.js#L3-L5